application/src/main/java/org/togetherjava/tjbot/commands/github/GitHubReference.java

```diff
@@ -130,13 +130,12 @@ protected MessageEmbed generateReply(GHIssue issue) throws UncheckedIOException
                 .map(this::getUserNameOrThrow)
                 .collect(Collectors.joining(", "));
 
+            Instant createdAt = issue.getCreatedAt().toInstant();
 
-            Calendar createdAt = Calendar.getInstance();
-            createdAt.setTime(issue.getCreatedAt());
-
-            // Format: 9 Oct, 2023
-            String dateOfCreation = "%d %s, %d".formatted(createdAt.get(Calendar.DATE),
-                    MONTHS.get(createdAt.get(Calendar.MONTH)), createdAt.get(Calendar.YEAR));
+            // Format: 09 Oct, 2023
+            DateTimeFormatter formatter =
```

Comment:
can create it as constant field

Reply:
Done, btw atm the channel pattern for `#issueNumber` is set to `general` which basically makes sure that it only works in `#general`. So i'm not sure which channel could be a good candidate for referencing issues. Maybe `#tj-bot`? or `#chit-chat`

Reply:
tjbot-discussions and server-suggestions